### PR TITLE
Package configs

### PR DESCRIPTION
**Pull request by daniel-sol**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,9 +1,46 @@
...
+test = ["pytest"]
+dev = ["pytest", "black", "flake8"]
+
+# [tool.setuptools]
+# package-dir = { "" = "src" }
+# include-package-data = true
+# platforms = ["any"]
+#
+# [tool.setuptools.packages.find]
+# where = ["src"]
```
Comment (equinor-ruaj) [inline pyproject.toml:37]: just remove completely if it works without these?

File: pyproject.toml [hunk 2]
```diff
@@ -1,9 +1,46 @@
...
+
+[bdist_wheel]
+universal = 1
+
+[aliases]
+test = "pytest"
+
+[project.scripts]
+sim2sumo = "fmu.sumo.sim2sumo.sim2sumo:main"
```
Comment (equinor-ruaj) [inline pyproject.toml:46]: think we need
```
[project.entry-points.ert]
fmu_sumo_sim2sumo_jobs = fmu.sumo.sim2sumo.hook_implementations.jobs
```

for ERT to work?

File: pyproject.toml (hunk 1 above)
Comment (daniel-sol) [reply pyproject.toml:37]: Removed most of it, but keeping tools.setuptools,packages.find to be explicit

File: pyproject.toml (hunk 2 above)
Comment (daniel-sol) [reply pyproject.toml:46]: Don't think it is required, has worked without it, but no harm having it

File: pyproject.toml [hunk 3]
```diff
@@ -1,9 +1,40 @@
...
 [tool.black]
 line-length = 79
+
+[project]
+name = "fmu-sumo-sim2sumo"
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+    "sumo-wrapper-python>=1.0.3",
+    "fmu-sumo-uploader",
```
Comment (equinor-ruaj) [inline pyproject.toml:19]: need `@ git+https://github.com/equinor/fmu-sumo-uploader.git@main` for `pip install .` to work

File: pyproject.toml [hunk 4]
```diff
@@ -1,9 +1,40 @@
...
+
+[tool.setuptools.packages.find]
+where = ["src"]
+
+
+[project.scripts]
+sim2sumo = "fmu.sumo.sim2sumo.sim2sumo:main"
+
+[project.entry-points.ert]
+fmu_sumo_sim2sumo_jobs = fmu.sumo.sim2sumo.hook_implementations.jobs
```
Comment (equinor-ruaj) [inline pyproject.toml:40]: missing `"`

File: pyproject.toml (hunk 3 above)
Comment (daniel-sol) [reply pyproject.toml:19]: Ok, I guess it is the right thing to do for the sake of installs, but when you want to develop, it is actually in the way. you have to reinstall with the -e flag afterwards for everything to work as intended.

File: pyproject.toml (hunk 4 above)
Comment (daniel-sol) [reply pyproject.toml:40]: Yes. done!

**Review by equinor-ruaj — APPROVED** [review]
lgtm

**Pull request merged**